package.json

```diff
@@ -9,6 +9,8 @@
   "scripts": {
     "start": "webpack --mode development",
     "dev": "webpack-dev-server --mode development --open",
+    "sass": "node-sass --include-path scss public/scss/main.scss public/css/style.css",
+    "watch-sass": "nodemon -e scss -x \"npm run sass & npm run dev\" ",
```

Comment:
Why `npm run dev` again here?

Reply:
Coz we have to run again after converting sass